src/Hl7.Fhir.Core.Tests/Introspection/FhirClassMappingTests.cs

```diff
@@ -34,5 +35,34 @@ bool isNested(Type testee)
                 return cm.IsNestedType;
             }
         }
+
+        [TestMethod]
+        public void TestPerformanceOfMappingAllClasses()
+        {
+            // just a random list of POCO types available in common
+            var typesToTest = ModelInfo.FhirCsTypeToString.Keys;
+            Console.WriteLine($"Creating a mapping for all {typesToTest.Count} pocos.");
+
+            var sw = new Stopwatch();
+            sw.Start();
+            for (int i = 0; i < 10; i++)
+                foreach (var testee in typesToTest)
+                    createMapping(testee);
+            sw.Stop();
+            Console.WriteLine($"No props: {sw.ElapsedMilliseconds/10.0}ms");
```

Comment:
Why are you dividing it by 10? It is already milliseconds, right?

Reply:
The test is run 10 times (see one line before)